torchaudio/prototype/io/streamer.py

```diff
@@ -0,0 +1,486 @@
+                chunk is dropped.
+
+            stream_index (int, optional): The source audio stream index.
+                If omitted, :py:attr:`default_audio_stream` is used.
+
+            filter_desc (str, optional): Filter description.
+                The list of available filters can be found at
+                https://ffmpeg.org/ffmpeg-filters.html
+                Note that complex filters are not supported.
+        """
+        i = self.default_audio_stream if stream_index is None else stream_index
+        torch.ops.torchaudio.ffmpeg_streamer_add_audio_stream(
+            self._s, i, frames_per_chunk, buffer_chunk_size, filter_desc
+        )
+
+    def add_video_stream(
+        self,
+        frames_per_chunk: int,
+        buffer_chunk_size: int = 3,
+        stream_index: Optional[int] = None,
+        filter_desc: Optional[str] = None,
+    ):
+        """Add output video stream
+
+        Args:
+            frames_per_chunk (int): A unit number of frames returned by Streamer as a chunk.
+                If the source stream is exhausted before enough frames are buffered,
+                then the chunk is returned as-is.
+
+            buffer_chunk_size (int): Internal buffer size.
+                When this many chunks are created, but
+                client code does not pop the chunk, if a new frame comes in, the old
+                chunk is dropped.
+
+            stream_index (int, optional): The source video stream index.
+                If omitted, :py:attr:`default_video_stream` is used.
+
+            filter_desc (str, optional): Filter description.
+                The list of available filters can be found at
+                https://ffmpeg.org/ffmpeg-filters.html
+                Note that complex filters are not supported.
+        """
+        i = self.default_video_stream if stream_index is None else stream_index
+        torch.ops.torchaudio.ffmpeg_streamer_add_video_stream(
+            self._s, i, frames_per_chunk, buffer_chunk_size, filter_desc
+        )
+
+    def remove_stream(self, i: int):
+        """Remove an output stream.
+
+        Args:
+            i (int): Index of the output stream to be removed.
+        """
+        torch.ops.torchaudio.ffmpeg_streamer_remove_stream(self._s, i)
+
+    def process_packet(self) -> int:
+        """Read the source media and process one packet.
+
+        The data in the packet will be decoded and passed to corresponding
+        output stream processors.
+
+        If the packet belongs to a source stream that is not connected to
+        an output stream, then the data are discarded.
+
+        When the source reaches EOF, then it triggers all the output stream
+        processors to ender drain mode. All the output stream processors
+        flush the pending frames.
+
+        Returns:
+            int:
+                ``0``
+                A packet is processed properly. The caller can keep
```

Comment:
To make it consistent with ``1`` and ``<0``
```suggestion
                A packet was processed properly. The caller can keep
```